src/main/java/org/opentripplanner/routing/edgetype/StreetEdge.java

```diff
@@ -223,16 +244,20 @@ public boolean canTraverse(VehicleDescription vehicle) {
         if (Objects.nonNull(vehicle)) {
             switch (vehicle.getVehicleType()) {
                 case MOTORBIKE:
-                    if (this.getMaxStreetTraverseSpeed() > MOTORBIKE_MAX_EDGE_TRAVERSE_SPEED_LOWER_BOUND) {
+                    if (this.getMaxStreetTraverseSpeed() > MotorbikeDescription.MAX_EDGE_TRAVERSE_SPEED_LOWER_BOUND) {
                         //If maximum traverse speed > 80 km/h, motorbikes are not allowed
                         return false;
                     }
                     break;
+                case KICKSCOOTER:
+                    if (getMaxStreetTraverseSpeed() > KickScooterDescription.MAX_EDGE_TRAVERSE_SPEED_LOWER_BOUND) {
+                        //If maximum traverse speed > 30 km/h, kickscooters are not allowed
+                        return false;
+                    }
```

Comment:
It seems that these changes alter the paths returned in response to the requests for some integration tests: 

ResponseItineraryTypeBasicIT.testBusAndKickscooter:98 
Expected: iterable containing ["WALK+KICKSCOOTER", "WALK+TRANSIT", "WALK+TRANSIT"]
but: item 0: was "WALK+TRANSIT"

ResponseItineraryTypeBasicIT.testKickscooter:73 » Processing Error reading ent...